js/task-02/ajax.js

```diff
@@ -0,0 +1,34 @@
+function onSubmit(){
+  
+  const url = "./ajax-handler.php";
```

Comment:
Nice. I was able to checkout your code and test it without any changes to your source.